crates/bevy_ui/src/render/mod.rs

```diff
@@ -277,7 +277,7 @@ pub fn extract_uinode_borders(
             Without<ContentSize>,
         >,
     >,
-    parent_node_query: Extract<Query<&Node, With<Parent>>>,
+    parent_node_query: Extract<Query<&Node>>,
```

Comment:
```suggestion
    node_query: Extract<Query<&Node>>,
```

Reply:
Well, since it's the query used to get the parent, it makes sense to be named "parent_node_query"

Reply:
I would prefer to name it based on what is is, not where it's used

Reply:
> I would prefer to name it based on what is is, not where it's used

agreed I think, but keep the name `parent_node` for the value retrieved from the query, where it is the actual Node component of the parent.